### ISSUE TITLE
Add linter and Travis test harness

### ISSUE THREAD

**Issue by chrisspen** [paradoxxxzero/gnome-shell-system-monitor-applet#348]
Add a static code analysis tool to check for formatting and syntax errors. Register this in Travis so pull requests get automatically tested.

**Comment by franglais125** [comment]
Nice, didn't know about this!

**Comment by chrisspen** [comment]
@paradoxxxzero, Could you please register this project under [Travis-CI](https://travis-ci.org/) so it can start processing my `.travis.yml` file?

**Comment by paradoxxxzero** [comment]
Sure: https://travis-ci.org/paradoxxxzero/gnome-shell-system-monitor-applet

**Comment by chrisspen** [comment]
Thanks.

**Comment by chrisspen** [comment]
I'm ready to merge this in. Does anyone have any final comments or suggestions?

**Comment by franglais125** [comment]
I just tested this on Gnome 3.22, it seems to work fine.

**Comment by chrisspen** [comment]
The only weird thing I saw was [this line](https://github.com/paradoxxxzero/gnome-shell-system-monitor-applet/blob/master/system-monitor%40paradoxxx.zero.gmail.com/extension.js#L843), which eslint doesn't like because it's an unused expression. However, when I removed it, the extension completely disappeared from display, so I kept it in and whitelisted it.

Any idea what the purpose of that line is?
